lab-lindsay/server.js

```diff
@@ -0,0 +1,58 @@
+
```

Comment:
**TODO FOR LOST POINTS**

This is a top-level comment that you forgot to fulfill these instructions:

for all request to / the server should respond with the following:

- a header containing Content-Type: text/plain
- a status code of 200
- a response with the string "hello world"